teambit.bvm/link/link.ts

```diff
@@ -105,11 +114,46 @@ function getBitBinPath(){
   return path.join('@teambit', 'bit', 'bin', 'bit');
 }
 
-function validateBinDirInPath(binDir: string){
+async function validateBinDirInPath(binDir: string, opts: { updatePath?: boolean } = { updatePath: true }) {
   const osPaths = (process.env.PATH || process.env.Path || process.env.path).split(path.delimiter);
-  if (osPaths.indexOf(binDir) === -1) {
+  if (osPaths.indexOf(binDir) !== -1) return;
+  if (!opts.updatePath) {
     const err = IS_WINDOWS ? windowsMissingInPathError(binDir, WINDOWS_INSTALL_TROUBLESHOOTING_DOCS_URL) : macLinuxMissingInPathError(binDir, MAC_LINUX_INSTALL_TROUBLESHOOTING_DOCS_URL);
     console.log(chalk.yellowBright(err));
+  } else {
+    const report = await addDirToEnvPath(binDir, {
+      overwrite: true,
+      position: 'end',
+      configSectionName: 'bit',
+    })
+    const output = renderSetupOutput(report)
+    if (output) {
+      console.log(output)
```

Comment:
don't console.log from the API layer, only from the command.
it's better to add all the info you need to the `LinkResult` type, and handle it in the command.
(I know there are failure messages in the API layer, we should move them to the command as well, but let's don't make the problem worse)